### PR TITLE
W3 BM - Strenger family + minor changes and fixes

### PR DESCRIPTION
**Pull request by TMLCJLK**
Ok, the initial proposal for the history files is done. From this point on the BM could be created without problems.

The only thing that could be considered as pending are:
- creating a CoA for the Strenger dynasty (Bloody Baron).
- making sure that landless cannon characters are where they should be.
- testing the balance for the scenario and adding initial levies if necessary (atm, Nilfgaard have a small disadvantage because of the number of vassals – and because the AI is too greedy to do something about it – they should probably start with a reasonable army ready for the war, but reinforcements will be a problem unless the player do something about it).
- keeping an eye for any unforeseen problem.

Of course, beyond that I could also (if it is of interest):
- populate the map with some basic count level characters (allowing players to play as different count level lords).
- work on older characters in order to allow earlier BMs in the future (just a thought).

### CODE REVIEW

**Comment by JonStryker** [comment]
I found a pic with the Polish place names: http://images.eurogamer.net/2014/pl/velen-rynsztunki-wsie.jpg
Since English language place names always seem a bit out of place in this Slavic world consider using these if you need additional place names. Wies means village, Dworek manor btw.

**Comment by TMLCJLK** [comment]
Nice. I will make the change.

File: MOD/Witcher/history/characters/020000_temerian.txt [hunk 1]
```diff
@@ -12488,3 +12500,42 @@
...
+
+## 93000 - 93099	Strenger (161)			/ Findetann
+9300 = {
+	name = "Phillip"		#Bloody Baron
+	dynasty = 161
+	dna=abcddsljaae
+	properties=ei0jbj0000000000
+	martial = 8 diplomacy = 5 stewardship = 5 intrigue = 4 learning = 4
+	religion=melitele culture=temerian
+	add_trait=canon add_trait=human_race add_trait=wroth add_trait=gluttonous add_trait=drunkard add_trait=stubborn add_trait=tough_soldier
```
Comment (rquinio) [inline MOD/Witcher/history/characters/020000_temerian.txt]: Nice traits :)

**Comment by rquinio** [comment]
Looking good !

**Pull request merged**